pkg/client/connector/install_actions.go

```diff
@@ -571,7 +571,7 @@ func (hcp *hideContainerPortAction) getPort(obj kates.Object, name string) (*kat
 			}
 		}
 	}
-	return nil, nil, fmt.Errorf("unable to locate port %s in container %s in %s %s.%s", hcp.PortName, hcp.ContainerName, objKind, obj.GetName(), obj.GetNamespace())
+	return nil, nil, fmt.Errorf("unable to locate port %s in container %s in %s %s.%s", name, hcp.ContainerName, objKind, obj.GetName(), obj.GetNamespace())
```

Comment:
Use the input for the log bc that's what we are checking in the logic and the provided input is different in 598 (hcp.PortName) than 618 + 627 (hcp.HiddenName)